examples/optimal_information_transport_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Example of shape-based image reconstruction
+using optimal information transformation.
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+import numpy as np
+import matplotlib.pyplot as plt
+from odl.util import snr
+from odl.deform import optimal_information_transport_solver
+import odl
+standard_library.install_aliases()
+
+
+I0name = './pictures/c_highres.png'
```

Comment:
We already have the submarine phantom in the utilities part. Why not using that one? If we want to add further phantoms, like the 2D letters C, I, V, ... and the 2D hand and brain images, that is where they should be added.